app/coffee/models/Doc.coffee

```diff
@@ -5,7 +5,8 @@ Schema = mongoose.Schema
 ObjectId = Schema.ObjectId
 
 DocSchema = new Schema
-	name          :     {type:String, default:'new doc'}
+	name          :     {type:String, default:'new doc'},
+        mimetype	:	{type:String, default:'application/x-tex'}
```

Comment:
we won't want to add this mimetype into the document ideally, does it ever change? If not can you just use mime.lookup(name) when it is required